packages/analytics/analytics-chart/src/locales/en.json

```diff
@@ -110,5 +110,15 @@
     "Hourly": "@timestamp per hour",
     "Daily": "@timestamp per day",
     "Weekly": "@timestamp per week"
+  },
+  "topNTable": {
+    "nameLabel": "Name",
+    "defaultEmptyStateTitle": "No data to display",
+    "defaultErrorStateTitle": "Something went wrong",
+    "errors": {
+      "meta": "Error: response must contain `meta`",
+      "display": "Error: `display` response incorrectly formatted (must have at least one key)",
+      "metricNames": "Error: `metricNames` response incorrectly formatted (must have at least one key)"
```

Comment:
issue: you should use single quotes `'` here instead of backticks; it can cause issues in some scenarios

Reply:
When would this cause issues?

Reply:
TBH I can't remember the scenario

Reply:
but I know we removed them (on purpose) from `en.json` in `khcp-ui` at some point